### DISCUSSION TITLE
User List UI in security not showing plus sign

### DISCUSSION THREAD

**Discussion by saurasingh** [apache/airflow#16864] [Q&A]
I have enabled RBAC in airflow.cfg and using OAUTH to enable single sign-on. Everything looks good except in the UI I am not seeing plus sign to modify the user Role or to add a new user. Currently, every user pops in as ADMIN and now I am not able to assign it to other roles. 

![Screen Shot 2021-07-08 at 10 07 05 AM](https://user-images.githubusercontent.com/19628707/124936155-4cd4b000-dfd4-11eb-8357-73df47640e50.png)

@potiuk Do you happen to know how I can fix this?

**Comment by potiuk** [comment]
I think this is a know issue with Flask Application Builder. See https://github.com/apache/airflow/issues/16202
